### ISSUE TITLE
xml parsing error: undefined entity when running the hello word example as desktop app

### ISSUE THREAD

**Issue by freemant2000** [flexxui/flexx#637]
When I run it, I get the following error:
![image](https://user-images.githubusercontent.com/1174319/79851713-43bfe900-83f8-11ea-8174-750f8fedb47a.png)

It works if launched in a browser though (despite that it always opens two tabs).

**Comment by almarklein** [comment]
Interesting. I cannot see where in our source we generate this code. Do you happen to use a title that includes this `&`symbol?

**Comment by freemant2000** [comment]
No. I haven't done anything beyond the hello world example. I suppose it is downloading some code from the local web server? How to manually download it or inspect the full file content?

**Comment by almarklein** [comment]
This error seems to be occurring before the browser app has even started up. 

What OS are you on?

As a workaround, you can always use a different runtime using [launch](https://flexx.readthedocs.io/en/stable/app/api.html#flexx.app.App.launch), e.g. `launch("browser")`, `launch("firefox-browser")`, `launch("chrome-app")`.
